werkzeug/wrappers.py

```diff
@@ -1233,9 +1233,9 @@ def get_wsgi_headers(self, environ):
         # Also update content_length accordingly so that the automatic
         # content length detection does not trigger in the following
         # code.
-        if 100 <= status < 200 or status == 204:
+        if 100 <= status < 200:
             headers['Content-Length'] = content_length = u'0'
-        elif status == 304:
+        elif status == 304 or status == 204:
```

Comment:
Yes, `x in (1, 2)` takes linear time, but `x in frozenset([1, 2])` also takes linear time (creating the frozenset from the list), plus the time to do the hashing / set membership check. The only case where a set is faster is when you store it so it's only created once and not every time you check.

Actually, I think in CPython it's also more efficient to use `x in (1, 2)` instead of e.g. `x in {1, 2}` - check the generated bytecode for the two expressions ;)